packages/mdc-textfield/index.js

```diff
@@ -136,10 +147,25 @@ class MDCTextField extends MDCComponent {
       registerTextFieldInteractionHandler: (evtType, handler) => this.root_.addEventListener(evtType, handler),
       deregisterTextFieldInteractionHandler: (evtType, handler) => this.root_.removeEventListener(evtType, handler),
       notifyIconAction: () => this.emit(MDCTextFieldFoundation.strings.ICON_EVENT, {}),
+      registerBottomLineEventHandler: (evtType, handler) => {
```

Comment:
I think it would still be nice to keep these methods in getBottomLineAdapterMethods_()

Reply:
talked offline. I'm going to leave it as is for now. Once we finish the rest of our "split up text field" PRs, we'll go back and see if this method is still readable.